nextflow.config

```
@@ -108,20 +108,23 @@ params {
     preprocessing_adapterremoval_qualitymax          = 41
 
     // Mapping
-    mapping_tool                = 'bwaaln'
-    mapping_bwaaln_n            = 0.01 // From Oliva et al. 2021 (10.1093/bib/bbab076)
-    mapping_bwaaln_k            = 2
-    mapping_bwaaln_l            = 1024 // From Oliva et al. 2021 (10.1093/bib/bbab076)
-    mapping_bwaaln_o            = 2 // From Oliva et al. 2021 (10.1093/bib/bbab076)
-    mapping_bwamem_k            = 19
-    mapping_bwamem_r            = 1.5
-    mapping_bowtie2_alignmode   = 'local'
-    mapping_bowtie2_sensitivity = 'sensitive'
-    mapping_bowtie2_n           = 0
-    mapping_bowtie2_l           = 20
-    mapping_bowtie2_trim5       = 0
-    mapping_bowtie2_trim3       = 0
-    mapping_bowtie2_maxins      = 500
+    mapping_tool                             = 'bwaaln'
+    mapping_bwaaln_n                         = 0.01 // From Oliva et al. 2021 (10.1093/bib/bbab076)
+    mapping_bwaaln_k                         = 2
+    mapping_bwaaln_l                         = 1024 // From Oliva et al. 2021 (10.1093/bib/bbab076)
+    mapping_bwaaln_o                         = 2 // From Oliva et al. 2021 (10.1093/bib/bbab076)
+    mapping_bwamem_k                         = 19
+    mapping_bwamem_r                         = 1.5
+    mapping_bowtie2_alignmode                = 'local'
+    mapping_bowtie2_sensitivity              = 'sensitive'
+    mapping_bowtie2_n                        = 0
+    mapping_bowtie2_l                        = 20
+    mapping_bowtie2_trim5                    = 0
+    mapping_bowtie2_trim3                    = 0
+    mapping_bowtie2_maxins                   = 500
+    mapping_circularmapper_elongation_factor = 500
+    mapping_circularmapper_elongated_fasta   = null
+    mapping_circularmapper_elongated_fai     = null
```

```suggestion
    mapping_circularmapper_elongationfactor = 500
    mapping_circularmapper_elongatedfasta   = null
    mapping_circularmapper_elongatedfai     = null
```

I poersonally would stick to hte `<section>_<tool>_<paramname>` convention here